### PR TITLE
Create a helper function for generating correctly typed Perseus test items

### PR DESCRIPTION
**Pull request by Myranae**
## Summary:
There are many places in webapp where custom types are created for Perseus items, frequently related to testing. This change adds a helper function that creates a generic Perseus test item that can have specific sections overridden with custom input. It will ensure tests are properly typed and help us know where updates need to be made if any changes happen to the types in Perseus.

Issue: LC-1658

## Test plan:
- Confirm all checks pass
- Test the new function in webapp with the npm snapshot
- In webapp, confirm related tests pass and related stories still work

### CODE REVIEW

File: packages/perseus/src/__tests__/test-utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    basicObject,
+    expected1,
+    questionCustomInfo,
+    expected2,
+    answerAreaCustomInfo,
+    hintsCustomInfo,
+    expected3,
+} from "./test-items/test-utils-items";
+
+describe("generateTestPerseusItem", () => {
```
Comment (Myranae) [inline packages/perseus/src/__tests__/test-utils.test.ts:15]: Would love suggestions on ways to make these tests more useful or additional tests that could be created for this change. :)

File: packages/perseus/src/util/test-utils.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * Generate a Perseus item object for testing.
+ *
+ * In order to better type Perseus objects used in testing, this function
+ * uses a basic Perseus object and updates it with custom values as needed.
+ *
+ * @param {Partial<PerseusItem>} customFields
+ * @returns PerseusItem
+ */
```
Comment (Myranae) [inline packages/perseus/src/util/test-utils.ts:37]: Is this useful here? I saw other functions had this information added, so wanted to include it. Seems handy to have a description.

File: packages/perseus/src/__tests__/test-items/test-utils-items.ts [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+                        randomize: true,
+                    },
+                    type: "radio",
+                },
+            },
+        },
+    ],
+};
+
+export const expected3: PerseusItem = {
```
Comment (handeyeco) [inline packages/perseus/src/__tests__/test-items/test-utils-items.ts:164]: Could we name these to be a little clearer? Maybe like `expectedExtendingHintsCase` and `expectedExtendingQuestionCase` or something like that?

**Review by handeyeco — CHANGES_REQUESTED** [review]
This looks good, very thorough tests.

Looking at `packages/perseus/src/util` I see that we have some tests next to the helpers. I think we should move these files a little bit:

```
packages/perseus/src/util
  - test-utils.ts
  - test-utils.test.ts
  - test-utils.testdata.ts
```

I can see why you put those there though, it seems like we have 3+ different ways of organizing things 😅 However I think tests are more discoverable when they're next to the files they test (esp when it comes to unit tests).

File: packages/perseus/src/__tests__/test-items/test-utils-items.ts (hunk 3 above)
Comment (Myranae) [reply packages/perseus/src/__tests__/test-items/test-utils-items.ts:164]: 100% On it :) Thanks for the suggestion!

Comment (Myranae) [reply packages/perseus/src/__tests__/test-items/test-utils-items.ts]: Updated the names and re-requested review

Comment (Myranae) [reply packages/perseus/src/__tests__/test-items/test-utils-items.ts]: Didn't see your comment about structure until now. Just refactored the test files.

File: packages/perseus/src/util/test-utils.test.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import {describe, it, expect} from "@jest/globals";
```
Comment (handeyeco) [inline packages/perseus/src/util/test-utils.test.ts:1]: I don't think these need to be explicitly imported anymore and should be removed.

**Review by handeyeco — APPROVED** [review]
Thanks!

**Comment by handeyeco** [comment]
Related to: https://github.com/Khan/webapp/pull/19775

**Pull request merged**